src/elm/Page/Register.elm

```diff
@@ -172,108 +151,197 @@ view guest model =
 
         name =
             accName model
-    in
-    if model.accountGenerated then
-        div [ class "main__register__details" ]
-            [ div [ class "register__congrats" ]
-                [ div [ class "register__woman" ] []
-                , p [ class "congrats__message" ]
-                    [ span [] [ text_ "register.account_created.congrats" ]
-                    , span [] [ text_ "register.account_created.account" ]
-                    , span [] [ text_ "register.account_created.created" ]
+
+        passphraseTextId =
+            "passphraseText"
+
+        passphraseInputId =
+            -- Passphrase text is duplicated in `input:text` to be able to copy via Browser API
+            "passphraseWords"
+
+        viewTitleForStep : Int -> Html msg
+        viewTitleForStep s =
+            let
+                step =
+                    String.fromInt s
+            in
+            p
+                [ class "py-4 mb-4 text-body border-b border-dotted text-grey border-grey-500" ]
+                [ text (tr "register.form.step" [ ( "stepNum", step ) ])
+                , text " / "
+                , strong
+                    [ class <|
+                        if s == 1 then
+                            "text-black"
+
+                        else
+                            "text-white"
                     ]
-                , div [ class "register__dog" ] []
+                    [ text_ ("register.form.step" ++ step ++ "_title") ]
                 ]
-            , div [ class "register__welcome" ]
-                [ text (tr "register.account_created.welcome_message" [ ( "username", name ) ]) ]
-            , div [ class "register__keys" ]
-                [ p [ class "key__title" ] [ text_ "register.account_created.twelve_words" ]
-                , p [ class "key", id "12__words" ] [ text (words model) ]
-                , p [ class "key__title" ] [ text_ "register.account_created.private_key" ]
-                , p [ class "key", id "p__key" ] [ text (privateKey model) ]
+    in
+    if model.accountGenerated then
+        div
+            [ class "flex-grow bg-purple-500 flex md:block"
+            ]
+            [ div
+                [ class "sf-wrapper"
+                , class "px-4 md:max-w-sm md:mx-auto md:pt-20 md:px-0 text-white text-body"
                 ]
-            , div [ class "register__instructions" ]
-                [ p [] [ text_ "register.account_created.instructions" ] ]
-            , button
-                [ onClick DownloadPdf
-                , class "btn btn__register"
+                [ div [ class "sf-content" ]
+                    [ viewTitleForStep 2
+                    , p
+                        [ class "text-xl mb-3" ]
+                        [ text_ "register.account_created.greet"
+                        , text " "
+                        , strong [] [ text name ]
+                        , text ", "
```

Comment:
what do you think about adding the spaces and commas to the translated text so we don't have to have those spacing stuff here?

Reply:
I think it's fine here.

The thing is that we should have the name in bold:
![image](https://user-images.githubusercontent.com/140053/81404423-171d0880-913e-11ea-84dc-00c52d655dcf.png)

So we have to separate `Hey`, `name`, and other text. Using the space at the end (`"Hey "`) is error-prone. The comma shouldn't be bold, but using it as the first symbol (`", it is the last..."`) looks odd in translations. So I decided to make the separation very obvious. It may look strange in Elm, but it's more convenient for translations.

Reply:
We could've used interpolation like `Hey <b>{{username}}</b>, ...` in the translation template, but we can't use HTML tags in there...